dev-python/m2crypto/m2crypto-0.31.0-r2.ebuild

```diff
@@ -25,7 +25,7 @@ RDEPEND="
 	libressl? ( dev-libs/libressl:0= )
 	virtual/python-typing[${PYTHON_USEDEP}]
 "
-DEPEND="${RDEPEND}
+BDEPEND="${RDEPEND}
```

Comment:
This does not seem right. openssl should be in DEPEND/RDEPEND, not BDEPEND.

I'm not sure about python-typing, but I suspect you don't need that in BDEPEND either.

Reply:
Thanks, yes, you were right.
I checked, and as far as I could see, no build file used `python-typing`. I modified this in the latest commit.